### PR TITLE
feat(gha): Add new gha for test for voice with postgres instance creation

### PR DESCRIPTION
**Pull request by mikepatt38**
Create a GHA workflow that will work with the new voice db

### CODE REVIEW

File: .github/workflows/test-integration-postgres-redis-rabbit.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: Reusable workflow for running postgres (with prisma) with rabbit and redis integration tests
```
Comment (Wagan8r) [inline .github/workflows/test-integration-postgres-redis-rabbit.yml:1]:
```suggestion
name: Reusable workflow for running postgres, rabbit, and redis integration tests
```
There's nothing "Prisma" about this workflow.

**Review by Wagan8r — CHANGES_REQUESTED** [review]
This whole workflow should be largely a copy/paste of the existing postgres workflows, including all of the `postgres_xyz` related and `setup_global_postgres`, `migration_command`, etc. inputs.

File: .github/workflows/test-integration-postgres-redis-rabbit.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        description: 'The command to run the integration tests'
+        default: 'npm run test:ci'
+        required: false
+        type: string
+      setup_global_postgres:
+        description: 'Setup Postgres @openphone/database repo'
+        default: true
+        required: false
+        type: boolean
+      global_postgres_ref:
```
Comment (Wagan8r) [inline .github/workflows/test-integration-postgres-redis-rabbit.yml:20]: I'd probably just remove this.

File: .github/workflows/test-integration-postgres-redis-rabbit.yml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    secrets:
+      token:
+        description: 'The GitHub/npm token'
+        required: true
+    inputs:
+      test_command:
+        description: 'The command to run the integration tests'
+        default: 'npm run test:ci'
+        required: false
+        type: string
```
Comment (Wagan8r) [inline .github/workflows/test-integration-postgres-redis-rabbit.yml:14]:
```suggestion
```
This is in here twice

**Review by Wagan8r — COMMENTED** [review]
This looks good, but waiting to approve until you test it. You can reference it in `voice` via:
```
  test-integration:
    uses: OpenPhone/gha/.github/workflows/test-integration-rabbit-redis.yml@ENG-7464
```
Once this is merged, you'll want to make a PR from `main` -> `v3`

**Review by bmatcuk — APPROVED** [review]
Works in my [repository pr](https://github.com/OpenPhone/voice/pull/711)

**Review by thesmo — APPROVED** [review]

**Review by Wagan8r — APPROVED** [review]

**Pull request merged**